build/npm.js

```diff
@@ -42,10 +42,15 @@ const npx = async (...args) => {
     await executeProcess("npx", ...args);
 };
 
-async function npmInstall(name) {
+async function npmInstall(name, offLine = true) {
```

Comment:
Nit: I'd expect _offline_ to default to `false`, not `true`. In `npm install`, there's a `--prefer-offline` flag, not a `--do-not-prefer-offline` flag, seems more intuitive to me to follow the same default behaviour.